.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+notifications:
+    email: false
+language: generic
+services:
+- docker
+
+arch:
+  packages:
+  - git
+  - go
+  script:
+  - make
```

Comment:
No need to run `make` first, `make test` should be fine by itself.

Reply:
Running `make` to test the building of `yay` which also leaves a `yay` executable ready for future system wide tests, as for what I checked, `make test` does not produce an executable

Reply:
`go test` will fail if the binary can't build, didn't think about leaving an executable behind though.